Makefile

```diff
@@ -27,11 +27,17 @@ cover:
   find tests -type d | xargs nosetests --with-coverage --cover-inclusive --cover-erase --cover-package=twilio
 
 docs-install:
-	. venv/bin/activate; pip install twilio sphinx
+	. venv/bin/activate; pip install -U sphinx
```

Comment:
Maybe better to use `pip install -r tests/requirements.txt`? It looks like we already require sphinx 1.4.9 there.